nemo/collections/common/tokenizers/en_ja_tokenizers.py

```diff
@@ -55,3 +39,19 @@ def normalize(self, text) -> str:
             return self.normalizer.normalize(text)
         else:
             return text
+
+
+class EnJaByteLevelProcessor:
```

Comment:
This isn't Ja specific. Can we just call it `ByteLevelProcesser` ?

Reply:
@mchrzanowski I think we should call this a `ByteLevelProcessor` and have it be the Processor of choice whenever the tokenizer_library is byte-level regardless of whether the language pair is English-Japanese.

Reply:
done. re-factored the src & target processor selection